backend/routes/PostReaction/Like.py

```
@@ -0,0 +1,79 @@
+import uuid  # Secure share token
+from typing import List
+from models.user import User
+from models.post import Post, PostMedia, PostDocument, Event, Like, Comment
+from zoneinfo import ZoneInfo
+from crud.notification import create_notification
+from schemas.notification import NotificationCreate
+from services.reaction import get_like_count, add_like, remove_like, notify_if_not_self, build_comment_response
+
+router = APIRouter()
+
+def get_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_db`

(ANN201)